src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+* {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+}
+
+.card {
+  font-family: "Roboto", "Arial", sans-serif;
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image-container {
```

Comment:
please use scss nesting here and everywhere possible.

Reply:
+